data/hai/hai culture conversations.txt

```diff
@@ -207,3 +207,47 @@ mission "Gambling Judgment"
+			`	Throughout this a handful of Hai have stopped to watch, and a couple of them let out small Hai chuckles at the conclusion before continuing on their way. Most of them simply roll their eyes and move on though. One child wanders too close as their parents steer around it and a little red holographic warning sign briefly flashes at them. No one seems keen to explain what you just saw and for the moment the crates remain resolutely closed.`
+			apply
+				set roboticconfusion
+			#This controls branching if there's a future follow-up where you get to see the performance.
+			`	Now that you think about it, you're not sure they all climbed back into the same boxes they came out of.`
+				decline
+			label translator
+			`	Music plays and some half dozen robotic caricatures of Hai in exaggerated dress spring forth from various crates and begin to caper about in a series of interactions. Your translator struggles to keep up  as they run through a rapid-fire string of presumably witty wordplay jokes that don't translate well and apparently improvisational humor. You realize that of course it can't actually be improvisational, but the illusion is good enough that the realization only catches up with you after the fact. The entire thing is set to a musical score, with beat rests at various points where the animatronic Hai keep turning away from each other to 'look' at the audience and wink. Holoprojected banners and glyphs flicker through a series of displays informing you that this is an advertisement for a fully robotic comedy musical called "The Grandmother's Rules". It claims to be complete with interactive heckling responses and provides a list of dates and venues, all in the future, where it will be showing.`
+			`	At the end each one of the robots strikes a pose and invites a different demographic to come along and "have the time of a decade", with ticket prices starting from 73 credits, before the entire cohort finish up and immediately clamber back into the crates. You're not sure they all climbed back into the same boxes they came out of.`
+			`	Throughout this a handful of Hai have stopped to watch, and a couple of them let out small Hai chuckles at the conclusion before continuing on their way. Most of them simply roll their eyes and move on though.`
+			`	One middle aged Hai with racoon-like grey around his eyes and wearing the type of attire typical of their merchant captains eyes you curiously and steps over to you. "Well well Captain, that's a remarkable box you have there. Didn't know they'd started making live translator devices! Did it keep up alright?"`
+			choice
+				`	"Oh it's the first of its kind."`
+					goto first
+				`	"I think I got the gist of it, though it only reads things out if I look directly at them."`
+			`	"Ah yes, I suppose that's a logical feature," he says. "I can only imagine it would be quite unwieldy if it tried to read out every piece of text in its vicinity.`
```

Comment:
```suggestion
			`	"Ah yes, I suppose that's a logical feature," he says. "I can only imagine it would be quite unwieldy if it tried to read out every conversation in its vicinity.`
```

Reply:
This is referring to when it translates text not spoken word using its cameras to identify context and content.

Reply:
Gotcha. Feel free to resolve!